app/src/main/java/org/kinecosystem/kinit/util/BindingUtils.java

```diff
@@ -160,4 +161,11 @@ public static void setVisibilityOn(EditText view, boolean visible) {
             view.setText("");
         }
     }
+
+    @BindingAdapter({"complete"})
+    public static void updateComplete(TransactionLayoutView layoutView, boolean transactionComplete) {
+        if (transactionComplete) {
+            layoutView.updateComplete();
```

Comment:
I think this should either stay inside TransactionLayoutView or changed into a more generic method where you call a generic action method depending on a specific boolean value.

Reply:
that was part of transactionlayout view but i moved it to here cause all binding methods are in single place

Reply:
In the same way you don't want to create a GenericUtils file that contains all private methods that a specific class wants to use - you don't want to add this specific binding method to BindingUtils. 
Either make a generic utility method or move it back to the specific class it serves.